README.adoc

```diff
@@ -124,7 +119,7 @@ include::finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.j
 
 Next, refer to `src/main/java/io/openliberty/guides/inventory/InventoryResource.java`.
```

Comment:
Can we change these instances where we point user to a file:

Next, refer to the `.java` file in `src/main/java/io/openliberty/guides/inventory/InventoryResource.java`
or 
Next, refer to the `src/main/java/io/openliberty/guides/inventory/InventoryResource.java` directory...

Some way to name what they are referring to? This is throughout the guide and I know we are usually pointing to a file, not a directory, so I wanted to discuss these instances.

Reply:
Ok, good one. We do want to name what we're referring to, my bad. Just I think .java file is too low level. I think "refer to the file" is good. I was to say "refer to the java file", but nearly all of our files are java files, there's no need to specify it.

Reply:
Ok--I will let you decide how you want to handle those. They certainly are not wrong the way they are.

Reply:
Ok, thanks a lot. 👍